Content.Tests/DMProject/Tests/Stdlib/Array/assign_blank.dm

```diff
@@ -0,0 +1,9 @@
+// RUNTIME ERROR
+
+/obj
+	var/thing[]
+
+/proc/RunTest()
+	var/obj/o = new
+	ASSERT(isnull(o.thing))
```

Comment:
This would lead to the test passing if `o.thing` isn't null, despite it _should_ being null.